blog_system/settings.py

```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-(rgt^grznqqrqa42q5f=gtji9cxo_*@+k%1&rk)bn9p9i_(7o7'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
DEBUG is set to True, which is not recommended for production environments as it can expose sensitive information. Ensure this is set to False in production.